kotlin-runtime/ftl-runtime/src/main/kotlin/xyz/block/ftl/logging/Logging.kt

```diff
@@ -29,18 +36,30 @@ class Logging {
       return logger(kClass.qualifiedName!!)
     }
 
-    fun init() {
-      val ple = PatternLayoutEncoder()
+    init {
+      val je = LoggingEventCompositeJsonEncoder()
+      je.context = logging.lc
 
-      ple.pattern = "%date %level %logger{10} - %msg%n"
-      ple.context = logging.lc
-      ple.start()
+      val providers: JsonProviders<ILoggingEvent> = je.providers
+      providers.setContext(je.context)
+      // Custom LogLevelJsonProvider converts level value to lowercase
+      providers.addProvider(object : LogLevelJsonProvider() {
+        override fun writeTo(generator: JsonGenerator, event: ILoggingEvent) {
+          JsonWritingUtils.writeStringField(generator, fieldName, event.level.toString().lowercase())
+        }
+      })
+      providers.addProvider(MessageJsonProvider())
+      je.providers = providers
+      je.start()
 
-      logging.appender.encoder = ple
+      logging.appender.target = ConsoleTarget.SystemErr.toString()
       logging.appender.context = logging.lc
+      logging.appender.encoder = je
       logging.appender.start()
 
-      logger(Logger.ROOT_LOGGER_NAME).level = Level.INFO
+      val rootLogger = logger(Logger.ROOT_LOGGER_NAME)
+      val rootLevelCfg = Level.valueOf(System.getenv("LOG_LEVEL") ?: DEFAULT_LOG_LEVEL)
```

Comment:
kotlin runtime will respect the LOG_LEVEL env variable - if none is provided the root logger defaults to info level at minimum, which suppresses the spammy GRPC debug logs